### DISCUSSION TITLE
creating style

### DISCUSSION THREAD

**Discussion by CFarhad** [mantinedev/mantine#4844] [Q&A]
i have to create style using hooks. if i can do it, what is the name of function or hook in mantine ?

this is an example
```jsx
let classes = createStyle({
  label:{
    color: red
  }
});

<Input styles={classes} />
```

**Comment by rtivital** [comment]
Starting from 7.0 Mantine no longer provides createStyles function, you can use Vanilla extract instead – https://mantine.dev/styles/vanilla-extract/
